workflows/phylorthology.nf

```diff
@@ -290,10 +291,10 @@ workflow PHYLORTHOLOGY {
     ch_filtered_ogs = FILTER_ORTHOGROUPS (
         INPUT_CHECK.out.complete_samplesheet,
         ch_orthogroups,
-        "4",
-        "4",
-        "1",
-        "2"
+        "3",
```

Comment:
I was gonna ask about this: why are these hard-coded? Can you add a comment explaining what these are doing?

Reply:
And should we parametrize this?

Reply:
Yeah, these should definitely be parameterized, as should a number of other things. 

I think that I'll make a list of what all would ideally be parameterized (across all modules) and then I'll create an issue listing these. Then, we can do a new PR focused on this?

Reply:
Let's create an issue for this @austinhpatton